airflow-core/src/airflow/plugins_manager.py

```diff
@@ -525,7 +525,7 @@ def integrate_macros_plugins() -> None:
         if plugin.name is None:
             raise AirflowPluginException("Invalid plugin name")
 
-        macros_module = make_module(f"airflow.macros.{plugin.name}", plugin.macros)
+        macros_module = make_module(f"airflow.sdk.macros.{plugin.name}", plugin.macros)
```

Comment:
```suggestion
        macros_module = make_module(f"airflow.sdk.definitions.macros.{plugin.name}", plugin.macros)
```

Though also: macros aren't definition time, so I wonder if this should actually be moved to:

```python
        macros_module = make_module(f"airflow.sdk.execution_time.macros.{plugin.name}", plugin.macros)
```

(and move the current `definitions.macros` to `execution_time.macros`

Reply:
Interesting call. Something worth thinking totally. 

Should we do it as a follow up to this task? Its related but not exactly!

Reply:
Will do in a follow up